### ISSUE TITLE
[COMPILE ERROR] Property 'subscriptionPeriod' not found on object of type 'SKProduct *'

### ISSUE THREAD

**Issue by Suraj-Tiwari** [dooboolab/react-native-iap#98]
### Version of react-native-iap
Latest
### Expected behavior
Successful Compile
### Actual behavior
Crash [While Compiling Issue is here file #ref](https://github.com/dooboolab/react-native-iap/blob/2fc12ca05cbddd7e3895a92104c6407ebff7c651/ios/RNIapIos.m#L227)
### Steps to reprodue the behabior
1) npm intall react-native-iap --save
2) Manual link IOS (Library & Binary file)
BUILD environment DEBUG compile. (Existing pod Project)

**Comment by dooboolab** [comment]
@Suraj-Tiwari I've just updated the readme and the published new version `0.3.1`. Could you follow the readme once more? @neilco have gave us [PR](https://github.com/dooboolab/react-native-iap/pull/99/commits/4e7d528694545ad9685a3dc78e39323cf576aa0f) which might help manual installation for pod project.

**Comment by neilco** [comment]
For anyone else who hits this issue, this is where the error originates:

```swift
  if (@available(iOS 11.2, *)) {
    itemType = product.subscriptionPeriod ? @"sub" : @"iap";
  }
```

Since the module is using [an API introduced in iOS 11.2](https://developer.apple.com/documentation/storekit/skproduct/2936884-subscriptionperiod), you'll need use **Xcode 9.2** or later in order to compile the iOS library.

**Comment by Suraj-Tiwari** [comment]
@neilco  yup you are right it was an xcode outdated version issue (as i was super lazy to udate :P ) Issue was fixed after i updated it.
